client/src/sections/resources/Banishers.tsx

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import Line from "../../components/Line";
+import Tile from "../../components/Tile";
+import { useHaveEquipped, useMyTurncount } from "../../hooks/useCall";
+import useGet from "../../hooks/useGet";
+import useHave from "../../hooks/useHave";
+import { returnBanishes } from "../../util/chunkBanish";
+import { $item, $skill } from "../../util/makeValue";
+import { plural } from "../../util/text";
+
+/**
+ * Class used to store banishers. Unless otherwise specified, all params are strings.
+ * @param name The name of the banisher source
+ * @param numberRemaining Number of the banishers remaining (#)
+ * @param access Boolean representing if the user can access the banish
+ * @param banishType Must be "skill", "item", or "equip"; kind of banish
+ * @param href (optional) The target for a link
+ * @param isEquipped (optional) Boolean of if the item needed is equipped
+ * @param equipItem (optional) String of the item you need to equip
+ * @param extraString (optional) An extra string to append to the end of the tile
+ */
+class BanisherObject {
+  name: string;
+  numberRemaining: number;
+  access: boolean;
+  banishType: string;
+  href: string;
+  banishedMonster: string;
+  isEquipped: boolean;
+  equipItem: string;
+  extraString: string;
+
+  constructor(
+    name: string,
+    numberRemaining: number,
+    access: boolean,
+    banishType: "skill" | "item" | "equip",
+    href?: string,
+    banishedMonster?: string,
```

Comment:
rather than submitting this through the constructor, why not initialize the weird banishedMonsters thing as a const at the top, and have this automatically parse it for you?